README.md

```diff
@@ -53,7 +55,7 @@ not straightforward.
 ## Very Much Frequently Asked Questions ❓
 
 ### How much doge can exist? – So many puppies! 🐕
-Early 2015 (approximately a year and a half after release) there were
+Early 2015 (approximately a year and a half after release) there will be
```

Comment:
This reads correct before this edit considering it's now past tense. It's worth reverting this hunk.

Reply:
Just realize my previous reply to your comment was placed in a wrong place. Basically I was saying, thank you for pointing it out. I copied the original README file from the master branch a few days back and the English text was "will be". I should have develop with dev-1.14.4 directly. Thanks!